Python/tests/syntax_test_python.py

```diff
@@ -2406,11 +2406,54 @@ def f(): pass
 #            ^^^^^^^ meta.sequence.tuple.python
 #            ^ punctuation.section.sequence.begin.python
 #                  ^ punctuation.section.sequence.end.python
-#                        ^^ punctuation.section.tuple.begin.python
-#                                ^ punctuation.section.tuple.end.python
-#                                    ^ punctuation.section.tuple.end.python
+#                        ^^ punctuation.section.sequence.begin.python
+#                                ^ punctuation.section.sequence.end.python
+#                                    ^ punctuation.section.sequence.end.python
 #                                           ^ punctuation.section.mapping.end.python
 
+unpack_ = [*(_ for [[][:], *_] in [((1, 2), "woops")])]
```

Comment:
I am legitimately surprised that this is *not* a syntax error, specifically allowing to assign something to `[]`. :roll_eyes: 
But well, neither is `[] = []` so it's at least consistent.